cookbook/bundles/best_practices.rst

```diff
@@ -190,13 +186,14 @@ the ``Tests/`` directory. Tests should follow the following principles:
 * The tests should cover at least 95% of the code base.
 
 .. note::
+
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
 Documentation
 -------------
 
-All classes and functions must come with full PHPDoc.
+All classes and functions must be fully documented using `phpDocumentor`_ tags.
```

Comment:
why phpDocumentor if Symfony relies on Sami and other PHPdoc generators are popular too?